NASA-AMMOS/aerie-ui
# [AERIE-1761] feat: derive duration from plan time bounds

This resolves https://jira.jpl.nasa.gov/browse/AERIE-1761 by adding a non-editable duration input field that displays the microsecond difference between the plan's specified start/end times as the duration format recently added (i.e. '2y 318d 6h 16m 19s 200ms 0us')

To test:
1. Go to the plan creation
2. Define a valid start/end time
3. Verify that the plan duration field below is updated to reflect the correct duration between the defined times in step 2
4. Verify that defining invalid times results in a "None" being displayed in the duration field
